frontend/packages/modelServing/src/components/projectDetails/NoModelsView.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import EmptyDetailsView from '@odh-dashboard/internal/components/EmptyDetailsView';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { ProjectObjectType, typedEmptyImage } from '@odh-dashboard/internal/concepts/design/utils';
+import { ModelServingPlatform } from '../../concepts/modelServingPlatforms';
+
+export const NoModelsView: React.FC<{ platform: ModelServingPlatform }> = ({ platform }) => (
+  <EmptyDetailsView
+    allowCreate
+    iconImage={typedEmptyImage(ProjectObjectType.modelServer)}
+    // imageAlt={isProjectModelMesh ? 'No model servers' : 'No deployed models'}
+    title={platform.properties.deployedModelsView.startHintTitle}
+    description={platform.properties.deployedModelsView.startHintDescription}
+    createButton={<Button>{platform.properties.deployedModelsView.deployButtonText}</Button>}
+    // footerExtraChildren={
+    //   deployingFromRegistry stuff
+    // }
+  />
+);
```

Comment:
Hmmm... I think we'll need to consider what the goals are for shared components like this... 

I wonder if these should be extensions consumed by the Project-area and then passed in... seems a bit funky if you want to do something special with your empty state.

Ideally we don't import the component from the base into it like this. Maybe we need to have some idea of shared components.

@christianvogt do you have thoughts on how to manage this?